person.rb

```diff
@@ -12,6 +12,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
Great job @Roland-Ntwali 🤝

- [ ] [Optional] Kindly ensure the method for adding rentals should be present in the Person class.